core-plugins/src/test/java/io/cdap/plugin/batch/joiner/JoinerTestRun.java

```diff
@@ -77,37 +82,55 @@ public void testInnerJoin() throws Exception {
      *
      */
 
+    ETLBatchConfig config = ETLBatchConfig.builder("* * * * *")
```

Comment:
this method is deprecated and shouldn't be used, there is a builder method that doesn't take any arguments.

Reply:
fixed.